yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-index-iops.psql

```diff
@@ -24,8 +30,7 @@ LEFT JOIN
     pg_index pi ON pi.indexrelid = c.oid
 WHERE
     n.nspname = ANY(ARRAY[string_to_array(:'schema_list', '|')])
-    AND (c.relkind = 'r' OR (c.relkind = 'i' AND NOT pi.indisprimary))
-    AND (psut.schemaname = n.nspname OR psi.schemaname = n.nspname);
+    AND (c.relkind IN ('r', 'p', 'i', 'I')) AND NOT (c.relkind IN ('i', 'I') AND pi.indisprimary);
```

Comment:
Just to confirm, this change will also include the partitioned table in index-iops metadata as well now but we will be filtering it completely while creating `table_index_stats`, right?

Reply:
yes exactly